infrastructure/validators/onboarding/run-validator/secure-node.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Secure Your Validator
+description: TODO
+---
+
+Validators in a Proof of Stake network are responsible for keeping the network in consensus and verifying state transitions. As the number of validators is limited, validators in the set have the responsibility to be online and faithfully execute their tasks.
+
+This primarily means that validators:
+
+- Must be high availability
+- Must have infrastructure that protects the validator's signing keys so that an attacker cannot take control and commit [slash-able behavior]()
```

Comment:
```suggestion
- Must have infrastructure that protects the validator's signing keys so that an attacker cannot take control and commit [slash-able behavior](TODO:update-path){target=\_blank}
```

Reply:
then we can do `control+f` and easily find all the pending routes :)